.circleci/config.yml

```diff
@@ -84,6 +84,8 @@ jobs:
           command: |
             npm run cypress start
             docker-compose run cypress npm run cypress db-seed
+            # Make sure the API key is the same so Percy snapshots are consistent
+            docker-compose -p cypress run postgres psql -U postgres -h postgres -c "update users set api_key = 'secret' where email ='admin@redash.io';"
```

Comment:
Can you instead make this be able to be set from db seeding?
```js
{
    route: '/setup',
    type: 'form',
    data: {
      name: 'Example Admin',
      email: 'admin@redash.io',
      password: 'password',
      org_name: 'Redash',
      api_key: 'secret', // <- HERE
    },
  },
```

Reply:
I don't like very much the idea of changing the API to this, in any case see my ideas in the other comment and LMK what you think.